shared/src/test/scala/spray/json/JsonParserSpec.scala

```diff
@@ -72,14 +72,6 @@ class JsonParserSpec extends Specification {
       val json = JsString("£0.99")
       JsonParser(json.prettyPrint.getBytes("UTF-8")) === json
     }
-    "be reentrant" in {
```

Comment:
ScalaJS doesn't implement `getClass.getResourceAsStream`. This test has been moved to the JVM-only project ("ParserSpecJvm.scala").